errors/base-error.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const ExtendableError = require('es6-error')
+const isObject = require('101/is-object')
+
+/**
+ * Extendable base class that includes additional information that effects the
+ * way errors are reported to rollbar.
+ * @class
+ * @author Ryan Sandor Richards
+ * @extends es6-error~ExtendableError
+ */
+module.exports = class BaseError extends ExtendableError {
+  /**
+   * Creates a new base error.
+   * @param {String} message Message for the error.
+   * @param {Object} data Additional data for the error to be given to rollbar.
+   * @param {Object} reporting Reporting options.
+   */
+  constructor (message, data, reporting) {
+    super(message)
+    this.data = isObject(data) ? data : {}
+    this.reporting = isObject(reporting) ? reporting : {}
```

Comment:
It seems the `reporting` object is not something you would really use, since you can just use the methods inside the constructor of the class.